crates/sui-core/src/consensus_adapter.rs

```diff
@@ -125,11 +134,23 @@ impl ConsensusAdapterMetrics {
                 authority_position_buckets.to_vec(),
                 registry,
             ).unwrap(),
-        }))
+            in_flight_semaphore_wait: register_int_gauge_with_registry!(
+                "in_flight_semaphore_wait",
```

Comment:
Should we give this metric a prefix, e.g. `consensus_adapter_` or `sequencing_`?

Reply:
I'll add `consensus_` prefix

Reply:
Actually you are right, `sequencing_` seem to be more common